src/components/collapsible.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode, FC, useState } from "react"
+import classNames from "classnames"
+
+import ArrowDown from "./icons/arrowDown"
+
+interface Props {
+  /**
+   * False if the element shall be expanded by default
+   * default: true
```

Comment:
Unless this changed in storybook 6 you don't need to provide the default here. It will be taken from the default values assignment in the component function.